android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java

```diff
@@ -38,14 +39,16 @@
 import com.google.android.gms.ads.LoadAdError;
 import com.google.android.gms.ads.admanager.AdManagerAdView;
 import com.google.android.gms.ads.admanager.AppEventListener;
+
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
 public class ReactNativeGoogleMobileAdsBannerAdViewManager
-    extends SimpleViewManager<ReactViewGroup> {
+  extends SimpleViewManager<LayoutWrapper> {
```

Comment:
ReactViewGroup overrides the [requestLayout](https://github.com/facebook/react-native/blob/0.66-stable/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java#L169), so extending `FrameLayout` which can only have a single child `BaseAdView` and setting our `requestLayout`